extensions/Variables/Variables.php

```diff
@@ -146,25 +146,25 @@ public static function getDir() {
 	# Parser Functions #
 	####################
 	
-    static function pf_varexists( Parser &$parser, $varName = '', $exists=true, $noexists=false ) {
+    static function pf_varexists( Parser $parser, $varName = '', $exists=true, $noexists=false ) {
         if( self::get( $parser )->varExists( $varName ) ) {
 			return $exists;
 		} else {
 			return $noexists;
 		}
     }
 	
-    static function pf_vardefine( Parser &$parser, $varName = '', $value = '' ) {
+    static function pf_vardefine( Parser $parser, $varName = '', $value = '' ) {
         self::get( $parser )->setVarValue( $varName, $value );
         return '';
     }
  
-    static function pf_vardefineecho( Parser &$parser, $varName = '', $value = '' ) {
+    static function pf_vardefineecho( Parser $parser, $varName = '', $value = '' ) {
         self::get( $parser )->setVarValue( $varName, $value );
         return $value;
     }
 		
-    static function pfObj_var( Parser &$parser, $frame, $args) {
+    static function pfObj_var( Parser $parser, $frame, $args) {
 		$varName = trim( $frame->expand( $args[0] ) ); // first argument expanded already but lets do this anyway
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)